src/software/ai/hl/stp/play/dynamic_plays/dynamic_play.cpp

```diff
@@ -24,9 +24,9 @@ void DynamicPlay::getNextTactics(TacticCoroutine::push_type &yield,
     }
 }
 
-void DynamicPlay::evaluate(double score)
+void DynamicPlay::terminate(const WorldPtr &world_ptr)
 {
-    support_tactic_success_scorer_->evaluate(score);
+    support_tactic_success_scorer_->evaluate(0);
```

Comment:
where would the score updates come?

Reply:
probably some reward function similar to GameplayMonitor, not sure yet